cluster/extensions/src/main/scala/com/lightbend/lagom/internal/cluster/projections/ProjectionRegistry.scala

```diff
@@ -76,9 +85,27 @@ private[lagom] class ProjectionRegistry(system: ActorSystem) {
 
   }
 
+  implicit val exCtx: ExecutionContext = system.dispatcher
+  implicit val timeout: Timeout        = Timeout(1.seconds)
+
   private[lagom] def getStatus(): Future[DesiredState] = {
-    implicit val exCtx: ExecutionContext = system.dispatcher
-    implicit val timeout: Timeout        = Timeout(1.seconds)
     (projectionRegistryRef ? GetDesiredState).mapTo[DesiredState]
   }
+
+  def stopWorker(projectionWorkerName: String): Future[Done] =
+    (projectionRegistryRef ? Stop(projectionWorkerName))
+      .mapTo[Done]
```

Comment:
Returning `Done` may give the impression that the worker has stopped when that `Future` completes. 

I see three possibilities: 

* document that that's not the case
* return some other type, eg: `Requested`. 
* return `Unit` to make explicit that this is a side-effect method and that this is a fire-and-forget